docs/Dockerfile

```diff
@@ -0,0 +1,23 @@
+FROM python:3.11
+
+ENV POETRY_VERSION=1.3.2 \
+    # make poetry install to this location
+    POETRY_HOME="/opt/poetry" \
+    # make poetry create the virtual environment in the project's root
+    # it gets named `.venv`
+    POETRY_VIRTUALENVS_IN_PROJECT=true \
```

Comment:
Maybe its better to avoid having poetry in this container at all and pin the deps

Reply:
@georgkrause I'm not sure how to do this, so I would appreciate help here.

Reply:
I gave some explanation in the chat yesterday:

* Solution one might be to configure poetry so that it does not create a venv. We are in a container already, there is no need for additional capsulation. [Docs](https://python-poetry.org/docs/configuration/#virtualenvscreate)
* Solution two might be to not use poetry at all and install the package using pip. 
 
I suspect solution two might has some downsides since we are not exactly building a proper python package here, so I'd say for now we only have to adjust the poetry config.

Reply:
I've updated this to install the dependencies using poetry without a venv, then build/run the container from the Makefile.